cloud-controller-manager/do/loadbalancers.go

```diff
@@ -733,13 +737,24 @@ func buildHTTP3ForwardingRule(ctx context.Context, service *v1.Service, godoClie
 		return nil, errors.New("certificate ID is required for HTTP3")
 	}
 
+	var targetProtocol string
+	if targetProtocol, err = getTargetProtocol(service); err != nil {
+		return nil, err
+	}
+	if targetProtocol == protocolTCP || targetProtocol == protocolUDP {
+		return nil, errors.New("target protocol annotation is not supported for TCP or UDP")
+	}
+	if targetProtocol == "" {
+		targetProtocol = protocolHTTP
```

Comment:
It looks like we don't need this anymore either given that `getTargetProtocol()` already returns `protocolHTTP` if the annotation is missing.